src/activitys/RaidEncounter.ts

```diff
@@ -0,0 +1,61 @@
+import { Metadata } from "main/types";
+import { raidEncountersById, VideoCategory } from "../main/constants";
+import Activity from "./Activity";
+
+/**
+ * Class representing a raid encounter.
+ */
+export default class RaidEncounter extends Activity {
+    private difficultyID: number = 0;
+    private encounterID: number = 0;
+
+    constructor(startDate: Date, 
+                category: VideoCategory, //wtf alex
```

Comment:
Yeah, wtf? :D Why wtf?

Reply:
not sure why I didn't just fix this, probably safe to assume RaidEncounter is VideoCategory.Raid...